README.md

```diff
@@ -22,14 +22,13 @@ The mini-lab is a small, virtual setup to locally run the metal-stack. It deploy
 - Linux machine with hardware virtualization support
 - kvm as hypervisor for the VMs (you can check through the `kvm-ok` command)
 - [docker](https://www.docker.com/) >= 20.10.13 (for using kind and our deployment base image)
-- [kind](https://github.com/kubernetes-sigs/kind/releases) == v0.20.0 (for hosting the metal control plane)
```

Comment:
Kind ist still required

Reply:
Tested without local kinD installation. Besides the `.kubeconfig` file created with `root` ownership, the setup still works.